app/scripts/templates/partial/user-card.mustache

```diff
@@ -3,7 +3,6 @@
   <div class="user-info">
     <div>
       <div class="prefillEmail">{{ email }}</div>
-      <a href="/" class="use-different" data-flow-event="use-different-account">{{#t}}Use a different email address{{/t}}</a>
```

Comment:
Are there other views the user-card are included in? If so, the new link will need to be added there.

Reply:
@shane-tomlinson no other view is using this user-card template.

Reply:
@shane-tomlinson no other view is using user-card template.